sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -4,10 +4,19 @@
 
 ### Features Added
 
+- Bumped up API version to `2021-04-10`.
+- Added support for encryption scope SAS (`ses` query parameter in SAS token).
+- Added encryption scope support for `BlobClient::CopyFromUri()`.
+- Added support for filtering blobs on container level.
+- Added support for tags copy mode (replace or copy from source) when copying blobs from URL.
+- Added support for permanent delete permission in SAS.
+
 ### Breaking Changes
 
 ### Bugs Fixed
 
+- Fixed listing blobs failure when blob name contains invalid characters in xml.
```

Comment:
How was this issue discovered?
Do we have a test for it. Also, can you please point me to where the fix is, in this PR, within the source?

Reply:
It's a REST API layer change. You'll see the change if you diff the swagger with the previous version.

Reply:
`BlobName` used to be a `std::string`. Now it's a struct with a `bool` in it indicating if it's encoded.

We'll decode and return a `std::string` in convenience layer, so that it's not a breaking change.